QuizBowlDiscordScoreTracker/discordToken.txt

```diff
@@ -0,0 +1 @@
+﻿NzY0NjExNjc1NTIwNzYxODk2.X4IyHQ.PN6QNO0zTiMaxbVy9kaC0zLVreI
```

Comment:
Remove this file and config.txt from the brnach. It should be tracked under .gitignore, but if it's not, we should update that file to exclude these two.
You should also rotate this secret; you can generate a new one from the Discord portal.

Reply:
such a stupid mistake... now it should be ok.